### ISSUE TITLE
Line breaks not respected

### ISSUE THREAD

**Issue by gysimichael** [swisstopo/swissgeol-boreholes-suite#1505]
1. Go to dev-boreholes.swissgeol.ch
2. Open a random borehole
3. Click on the Lithology tab
4. Create a new lithology description
5. Typ the discription with line breaks

**Expected behavior**
Line breaks are respected in the view mode

**Current behavior**
Line breaks are not respected in the view mode

![Image](https://github.com/user-attachments/assets/fb25e868-5623-4622-8d1a-188851191fdb)

![Image](https://github.com/user-attachments/assets/644f279c-867c-4fb8-a392-882588c1bfe4)

This is also valid for facies description

**Comment by gysimichael** [comment]
wir haben ein overflow, wenn zu viele Zeichen in der Bemerkung eingegeben werden, sollte in linebreak machen

<img width="800" height="170" alt="Image" src="https://github.com/user-attachments/assets/a590f764-c2aa-4a1f-bdcf-d8ac7d7e98d6" />

**Comment by gysimichael** [comment]
die Line breaks werden aus meiner Sicht noch nicht korrekt dargestellt:

1111
2222
3333
4444
5555

<img width="1896" height="924" alt="Image" src="https://github.com/user-attachments/assets/50a6bd66-470e-4ab5-92af-186a1aa46093" />

**Comment by RBordoni** [comment]
Im Bearbeitungsmodus kommt es richtig. Nach dem "Bearbeitung beenden" ist es bei der Lithologischen Beschreibung nicht umgebvrochen.

In Bearbeitung:

<img width="999" height="219" alt="Image" src="https://github.com/user-attachments/assets/ddda3cd8-60ab-4a7f-9bc4-6a798960979f" />

Nach "Bearbeitung beenden":

<img width="1375" height="277" alt="Image" src="https://github.com/user-attachments/assets/5b647300-3c05-4c94-8cb1-51c93581bc9a" />

**Comment by tschumpr** [comment]
@RBordoni Ich habs im Issue für die View-Tabelle vermerkt. Die muss sowieso noch überarbeitet werden.

**Comment by stijnvermeeren-swisstopo** [comment]
Im Editor-Sicht funktionieren die Line Breaks korrekt für mich auf DEV.

Im View-Ansicht noch nicht, aber wie von @tschumpr erwähnt, wird das noch in #2171 gefixt.

Damit kann dieses Issue aus meiner Sicht geschlossen werden.

Doch noch eine kleine Sache (jedoch nicht direkt in Zusammenhang mit den Line Breaks): das Inputfeld heist im Moment "Bemerkungen" (sowohl bei der lithologischen wie bei der faziellen Beschreibung). Aus meiner Sicht wäre "Beschreibung" richtiger. So ist es auch in den Designs auf Figma benannt: https://www.figma.com/design/cEiOoOazAQZqpRY92ZhBeO/swissgeol?node-id=14880-88357&t=zsHm8doGud86Pk5q-0

<img width="1215" height="595" alt="Image" src="https://github.com/user-attachments/assets/079fa9d2-7488-4096-b744-b5db32329a71" />

**Comment by tschumpr** [comment]
Die angepasste View inkl. Line Break ist jetzt auf DEV:

<img width="2082" height="788" alt="Image" src="https://github.com/user-attachments/assets/fe7973f6-6e54-488c-bf8b-f989e4a68bd2" />

**Comment by tschumpr** [comment]
> Doch noch eine kleine Sache (jedoch nicht direkt in Zusammenhang mit den Line Breaks): das Inputfeld heist im Moment "Bemerkungen" (sowohl bei der lithologischen wie bei der faziellen Beschreibung). Aus meiner Sicht wäre "Beschreibung" richtiger. So ist es auch in den Designs auf Figma benannt: https://www.figma.com/design/cEiOoOazAQZqpRY92ZhBeO/swissgeol?node-id=14880-88357&t=zsHm8doGud86Pk5q-0

@stijnvermeeren-swisstopo Ich kann das gerne noch anpassen. Soll es auch in der Lithologie Beschreibung werden oder behalten wir da Bemerkung als Label?

**Comment by stijnvermeeren-swisstopo** [comment]
@tschumpr Gemäss Design soll das Feld sowohl bei der lithologischen wie auch bei der faziellen Beschreibung als "Beschreibung" gelabelt werden.

<img width="1186" height="542" alt="Image" src="https://github.com/user-attachments/assets/5ee48256-ab2a-49bb-aa1c-41aab33834a5" />

<img width="1201" height="644" alt="Image" src="https://github.com/user-attachments/assets/0f7805aa-934c-4dc6-8d70-c8c4e122c36d" />

**Comment by tschumpr** [comment]
@stijnvermeeren-swisstopo Ja, das hab ich so angepasst. Meine Frage war, ob diese Bezeichnung auch in der Lithologie Sinn macht, aber ich habs jetzt da gelassen. Ich hab auch noch den Box-Titel für Lithologische Beschreibung von _Grunddaten_ zu _Beschreibung_ geändert, damits mit Figma übereinstimmt. (Änderungen sind noch nicht gemerged)

**Comment by stijnvermeeren-swisstopo** [comment]
@tschumpr Bei der Bearbeitung der Lithologie sind die Labels bereits korrekt, dort gibt es tatsächlich "Bemerkungen".
